### PR TITLE
Change the type of Request's request_timeout_sec to float

### PR DESCRIPTION
**Pull request by jkozlowicz**

### CODE REVIEW

**Review by leandro-lucarella-frequenz — COMMENTED** [review]
LGTM but I'm waiting for the approval until we have the `RELEASE_NOTES`, so you can add a note about this change.

**Comment by leandro-lucarella-frequenz** [comment]
OK, `RELEASE_NOTES.md` is ready, can you write an entry for this change?

**Comment by jakub-toptal** [comment]
> OK, `RELEASE_NOTES.md` is ready, can you write an entry for this change?

Done

File: RELEASE_NOTES.md
Comment (leandro-lucarella-frequenz) [inline RELEASE_NOTES.md]:
```suggestion
* `PowerDistributor` - type of `Request`'s `request_timeout_sec` has been changed from `int` to `float`
```

**Review by leandro-lucarella-frequenz — DISMISSED** [review]
Super nitpick, so all good with going as is! :)

File: RELEASE_NOTES.md
Comment (jakub-toptal) [reply RELEASE_NOTES.md]: done

**Review by leandro-lucarella-frequenz — APPROVED** [review]

**Pull request merged**
